GuaranteedRate.Sextant/Config/JsonEncompassConfig.cs

```
@@ -156,5 +157,24 @@ public IEncompassConfig GetConfigGroup(string key)
 
             return new JsonEncompassConfig(val.ToString());
         }
+
+        /// <summary>
+        /// Loads the config from the give string.  Useful for testing.
+        /// </summary>
+        /// <param name="configAsString">The json string</param>
+        /// <returns></returns>
+        public bool LoadFromString(string configAsString)
+        {
+            try
+            {
+                _jsonObject = JObject.Parse(configAsString);
+                return true;
+
+            }
+            catch (Exception)
+            {
+                throw;
```

Same questions about the empty catch/throw and returning false